Gruntfile.js

```diff
@@ -618,7 +618,7 @@ module.exports = function(grunt) {
         ],
       },
       'webapp-js': {
-        files: ['webapp/src/js/**/*', 'shared-libs/**'],
+        files: ['webapp/src/js/**/*', 'shared-libs/*/src/**/*'],
```

Comment:
This was super annoying cause the watcher when insane when tests did an `npm ci` for every shared lib (the watcher watched `node_modules` as well)